src/library_fs.js

```diff
@@ -1,6 +1,10 @@
 mergeInto(LibraryManager.library, {
   $FS__deps: ['$ERRNO_CODES', '$ERRNO_MESSAGES', '__setErrNo', '$PATH', '$TTY', '$MEMFS', '$IDBFS', '$NODEFS', 'stdin', 'stdout', 'stderr'],
   $FS__postset: 'FS.staticInit();' +
+                'Module["FS"] = Object.create(null);' +
```

Comment:
>Let's put it on the FS object itself

You're right, that's better. Fixed.

>Side note, why not do {} instead of Object.create(null)

To make `in` work without excesses (e.g. `'toString' in FS.filesystems`, `'__proto__' in FS.filesystems`). Though I'm ok with the current variant too.